src/runtime-rs/crates/runtimes/common/src/message.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::Result;
+use tokio::sync::mpsc::{channel, Receiver, Sender};
+
+#[derive(Debug)]
+pub enum Action {
+    StartListen,
+    StopListen,
+    ShutdownForce,
```

Comment:
- Why is there no `Shutdown`?
- What does "force" imply here? Is any data lost?

Reply:
Thanks for your review, change to `Shutdown`, using shutdown is more clear.